frontend/src/v4/modules/viewpoints/viewpoints.sagas.ts

```diff
@@ -220,7 +220,7 @@ export function* showViewpoint({teamspace, modelId, view, ignoreCamera}) {
 
 export function* fetchViewpointGroups({teamspace, modelId, view}) {
 	try  {
-		const groupsObject = getViewpointWithGroups({teamspace, modelId, view});
+		const groupsObject = yield getViewpointWithGroups({teamspace, modelId, view});
```

Comment:
First time I click on the view, the camera repositions correctly, but the groups are not highlighted/selected

https://github.com/user-attachments/assets/a4f033b9-bffa-458c-8f2c-bc1df6e7c96d